.env

```diff
@@ -1 +1,3 @@
 DATABASE_URL=postgres://agnes:123@localhost:5432/papel
+
+USER_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZW1haWwiOiJhZ25lc0BnbWFpbC5jb20iLCJ0eXBlIjoiQ0xJRU5UIiwiaXNfYWRtaW4iOmZhbHNlLCJpYXQiOjE1ODIxODgzNDR9.wBobIwZDqEHNwXxdThrn2KIchtGJYjKGGJWNNU1cxZc
```

Comment:
why do you keep pushing your .env file. This is a secret file. No one should be able to access it. Add it to your .gitignore file.